HappyCamps/src/app/services/api.service.ts

```diff
@@ -0,0 +1,17 @@
+import { HttpClient } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class ApiService {
+
+  private getAllUrl = 'https://localhost:7027/GetAll'
```

Comment:
make this a abse url and add the extension "getAll" to another variable or concatenate to the request itself

Reply:
This was created only for testing purpose only. Will be deleted in the upcoming commits.